CHANGELOG.md

```diff
@@ -3,6 +3,10 @@
 This file documents all notable changes to Ambassador Helm Chart. The release
 numbering uses [semantic versioning](http://semver.org).
 
+## v6.2.3
```

Comment:
Might want to mention the changes for #47 here as well...

Reply:
Thanks for the catch!